src/device/wg_controller.rs

```diff
@@ -200,12 +199,60 @@ async fn consolidate_wg_peers<
     for key in delete_keys {
         telio_log_info!("Removing peer: {:?}", actual_peers.get(key));
         wireguard_interface.del_peer(*key).await?;
+
+        // Remove from session_keeper
+        if let Some(sk) = session_keeper {
+            sk.remove_node(key).await?;
+        }
     }
 
+    let batcher_threshold = features
+        .batching
+        .as_ref()
+        .map(|b| Duration::from_secs(b.direct_connection_threshold as u64));
```

Comment:
Threshold should be configurable for each type, as each persistent-keepalive can be different, threshold cannot be a constant shared through all of them

Reply:
We decided to reuse this value for now